microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -152,6 +153,14 @@ When these domain lists are in place, Microsoft Edge no longer shows the additio
 Eventually, the `url_handlers` feature will be replaced by [`scope_extensions`](https://github.com/WICG/manifest-incubations/blob/gh-pages/scope_extensions-explainer.md), but that spec is still in development.  `scope_extensions` will produce the same result as `url_handlers`.
 
 
+<!-- ====================================================================== -->
+## Make updates to your app
+
+Generally, when you update your PWA code, you don't need to create a new app package and submit it to the Microsoft Store again. For example, when you make changes to your app's front-end HTML, CSS, and JavaScript code, or to your service worker, the changes are downloaded by the browser that's running your app from your web server the next time the app is launched.
+
+However, if you make changes to the web app manifest file, for example if you change the app icon or name, or if you change or add manifest members like `file_handlers`, `protocol_handlers`, or `share_handlers`, then you must create a new app package and submit it to the Microsoft Store.  This is because the information found in the web app manifest file is copied to the Windows app package for better integration with Windows.
```

Comment:
```suggestion
However, if you make changes to the web app manifest file, such as changing the app icon or app name, or changing manifest members such as `file_handlers`, `protocol_handlers`, or `share_handlers`, you must create a new app package and submit it to the Microsoft Store.  This is because the information found in the web app manifest file is copied to the Windows app package for better integration with Windows.
```